src/Tasks/Microsoft.Common.CurrentVersion.targets

```
@@ -1824,14 +1824,18 @@ Copyright (C) Microsoft Corporation. All rights reserved.
         much information as possible will be passed to the compilers.
         -->
 
+    <PropertyGroup>
+      <MSBuildGlobalPropertiesToRemoveFromProjectReferences Condition="'$(DoNotPassOutputPathToReferencedProjects)'=='true'">;OutputPath</MSBuildGlobalPropertiesToRemoveFromProjectReferences>
```

Prefer positive names, so you don't have quite so much confusion about things like `!DoNot...`

```suggestion
      <MSBuildGlobalPropertiesToRemoveFromProjectReferences Condition="'$(PassOutputPathToReferencedProjects)'=='false'">;OutputPath</MSBuildGlobalPropertiesToRemoveFromProjectReferences>
```

---

Should this append to list rather than overwrite?

---

As I recall, this is a novel property, so there's nothing to append to. If I'm misremembering, I can fix that.

---

That's true today--but there's no way to ensure that only we use it, so make it less of a footgun for others to use.